### PR TITLE
init

### PR DESCRIPTION
**Pull request by Zheguisoft**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/58190)

### CODE REVIEW

File: manifests/z/Zheguisoft/IpchatGuiPc/1.0.0/Zheguisoft.IpchatGuiPc.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Publisher:          Zheguisoft
+PackageName:        ipchat gui pc
+License:            Zheguisoft
+ShortDescription:   This is a chat tool
+Installers: 
+ - Architecture:    x64
+   InstallerType:   exe
+   InstallerUrl:    https://github.com/Zheguisoft/ipchat/ipchat_gui_pc.exe
+   InstallerSha256: 0bcc40e480184a22d707bbaab7bbbac7ba885676b95638cc9799ca6e18ccf729
+ManifestType:       singleton
```
Comment (Trenly) [inline manifests/z/Zheguisoft/IpchatGuiPc/1.0.0/Zheguisoft.IpchatGuiPc.yaml:13]: Singleton type manifests are not preferred; I suggest use tools like [wingetcreate](/microsoft/winget-create) or [YamlCreate](/microsoft/winget-pkgs/tree/master/doc/tools/YamlCreate.md) to create the manifest

File: manifests/z/Zheguisoft/IpchatGuiPc/1.0.0/Zheguisoft.IpchatGuiPc.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+PackageIdentifier:  Zheguisoft.IpchatGuiPc
+PackageVersion:     1.0.0
+PackageLocale:      zh-CN
+Publisher:          Zheguisoft
+PackageName:        ipchat gui pc
+License:            Zheguisoft
```
Comment (Trenly) [inline manifests/z/Zheguisoft/IpchatGuiPc/1.0.0/Zheguisoft.IpchatGuiPc.yaml:6]: A username isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate

File: manifests/z/Zheguisoft/IpchatGuiPc/1.0.0/Zheguisoft.IpchatGuiPc.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+PackageIdentifier:  Zheguisoft.IpchatGuiPc
+PackageVersion:     1.0.0
+PackageLocale:      zh-CN
+Publisher:          Zheguisoft
+PackageName:        ipchat gui pc
+License:            Zheguisoft
+ShortDescription:   This is a chat tool
```
Comment (Trenly) [inline manifests/z/Zheguisoft/IpchatGuiPc/1.0.0/Zheguisoft.IpchatGuiPc.yaml:7]: Please provide a better description

**Review by Trenly — CHANGES_REQUESTED** [review]
Please remove the .exe file from the PR.
Also, I do not believe this application can install silently. Silent install is required for packages to be added to the repository
image
This also appears to have a dependency on Java. Dependencies are not yet supported -
image

**Comment by Trenly** [comment]
@Zheguisoft You cannot host the release file in the winget repository. It needs to be in a separate repository or in a different hosting solution like an amazon S3 bucket. 

When you upload it to your fork, it causes it to be added to the PR

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/58190#pullrequestreview-946519316

**Comment by Zheguisoft** [comment]
What does it mean? How can I fix it?

File: manifests/z/Zheguisoft/IpchatGuiPc/1.0.0.0/Zheguisoft.IpchatGuiPc.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Zheguisoft.IpchatGuiPc
+PackageVersion: 1.0.0.0
+PackageLocale: en-US
+Publisher: Zheguisoft
```
Comment (ItzLevvie) [inline manifests/z/Zheguisoft/IpchatGuiPc/1.0.0.0/Zheguisoft.IpchatGuiPc.locale.en-US.yaml:7]:
```suggestion
Publisher: Zheguisoft, Inc.
```
This must match Control Panel.

File: manifests/z/Zheguisoft/IpchatGuiPc/1.0.0.0/Zheguisoft.IpchatGuiPc.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Zheguisoft.IpchatGuiPc
+PackageVersion: 1.0.0.0
```
Comment (ItzLevvie) [inline manifests/z/Zheguisoft/IpchatGuiPc/1.0.0.0/Zheguisoft.IpchatGuiPc.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.8
```
This must match Control Panel.

File: manifests/z/Zheguisoft/IpchatGuiPc/1.0.0.0/Zheguisoft.IpchatGuiPc.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Zheguisoft.IpchatGuiPc
+PackageVersion: 1.0.0.0
```
Comment (ItzLevvie) [inline manifests/z/Zheguisoft/IpchatGuiPc/1.0.0.0/Zheguisoft.IpchatGuiPc.yaml:5]:
```suggestion
PackageVersion: 1.8
```
This must match Control Panel.

File: manifests/z/Zheguisoft/IpchatGuiPc/1.0.0.0/Zheguisoft.IpchatGuiPc.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Zheguisoft.IpchatGuiPc
+PackageVersion: 1.0.0.0
```
Comment (ItzLevvie) [inline manifests/z/Zheguisoft/IpchatGuiPc/1.0.0.0/Zheguisoft.IpchatGuiPc.installer.yaml:5]:
```suggestion
PackageVersion: 1.8
```
This must match Control Panel.

File: manifests/z/Zheguisoft/IpchatGuiPc/1.0.0.0/Zheguisoft.IpchatGuiPc.locale.en-US.yaml [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+
+PackageIdentifier: Zheguisoft.IpchatGuiPc
+PackageVersion: 1.0.0.0
+PackageLocale: en-US
+Publisher: Zheguisoft
+PublisherUrl: https://www.zheguisoft.com
+PublisherSupportUrl: https://www.zheguisoft.com/zh-cn/support.htm
+Author: Zheguisoft
+PackageName: ipchat_gui_pc
+License: MIT License
```
Comment (ItzLevvie) [inline manifests/z/Zheguisoft/IpchatGuiPc/1.0.0.0/Zheguisoft.IpchatGuiPc.locale.en-US.yaml:12]:
```suggestion
License: MIT
```

File: manifests/z/Zheguisoft/IpchatGuiPc/1.0.0.0/Zheguisoft.IpchatGuiPc.locale.en-US.yaml [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+Copyright: Shanghai Zhegui Software Co., Ltd.
+ShortDescription: This is a chat tool, similar to IpMSg
+Description: This is a chat tool, similar to IpMSg
+ReleaseNotes: 1.0.0.0
```
Comment (ItzLevvie) [inline manifests/z/Zheguisoft/IpchatGuiPc/1.0.0.0/Zheguisoft.IpchatGuiPc.locale.en-US.yaml:16]:
```suggestion
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**